boards/rcar_salvator_xs_m3.overlay

```diff
@@ -19,6 +19,14 @@
 };
```

Comment:
Please remove  - qemu_cortex_a53 mention from the commit message

Reply:
Removed